### DISCUSSION TITLE
Add audio volume control within the application

### DISCUSSION THREAD

**Discussion by Porkepix** [wireapp/wire-desktop#5076] [Ideas]
It would be a nice addition within the application to be able to have audio volume control, both in the form of controlling global volume and individual audio level for every participant in an audio call (this can include adjusting audio levels if the sound is loud for some and very light for some others, or muting someone that let mic one with noises and so on).

**Comment by ffflorian** [comment]
Hi @Porkepix, good idea! I added a feature request label and will notify you here if we implement this at some point.

**Comment by mastercoms** [comment]
I would really enjoy this feature, AGC is not enough
